### DISCUSSION TITLE
Sprint 5 Goal

### DISCUSSION THREAD

**Discussion by timblegoorn** [cs481-ekh/f22-kitty-cat-studios#360] [Sprint Planning]
AI Team Sprint Goal: 

TBD (will have team meeting 11/10 to formalize sprint goals)

Stretch goals:  

TBD

**Comment by CameronSewell** [comment]
Sprint 5 planning meeting 11/10/2020

Team Leader: Cameron Sewell
Scribe: Annika

Present: Nathan, Tavi, Cameron
Unexcused absent: Annika


Team Goal: Finish the new runners and tie them to the selection menu, further improve the SUB map and add obstacles.

Nathan: Design the SUB Day level obstacles and AI and player spawning
Tavi: Finishing runner materials
Cameron: add models to runner selection, night level obstacles, adding ai spawning, and level spawning
Annika: Extensive testing on the new features, verifying any bugs reported in issues/adding any new bugs, once verified trying to fix as many as possible.

We should work on fixing bugs and going through technical debt as well.
Cameron will reach out to Ben soon to set up our project handoff meeting.

**Comment by camronCollinsworth** [comment]
Sprint 5 planning meeting 11/10

Product Owner: Damian
Scribe: Camron

AI Team Sprint Goals:

-  Fix bugs present in the game, targeting bugs that cause extreme gameplay abnormalities and crashes
- Improve AI movement system to conform with technical requirements of the game
- Streamline game loop and menu system to better include new game modes and options

Stretch Goal:
- implement wave survival game mode
